RELEASE.md

```diff
@@ -14,7 +14,47 @@ Further details on building, testing and use can be found in [README.md](https:/
 
 ## Release notes
 
-This is version 0.7.1-dev of oqs-provider which continues from the earlier 0.7.0 release. This release is fully tested to be used in conjunction with the main branch of [liboqs](https://github.com/open-quantum-safe/liboqs) and is guaranteed to be in sync with v0.12.0 of `liboqs` as and when released.
+This is version 0.8.0-rc1 of oqs-provider which continues from the earlier 0.7.0 release. This release is fully tested to be used in conjunction with the main branch of [liboqs](https://github.com/open-quantum-safe/liboqs) and is guaranteed to be in sync with v0.12.0 of `liboqs` as and when released.
+
+### Deprecation notice
+
+The associated liboqs v0.12.0 release will be the last liboqs release to include Kyber (that is, the NIST Round 3 version of Kyber, prior to its standardization by NIST as ML-KEM in FIPS 203). Applications should switch to ML-KEM (FIPS 203).
```

Comment:
Why is this phrased in `liboqs` terms? This is `oqsprovider` and the ramifications for _its_ users should be documented, e.g., "This is to notify users of *kyber* algorithms to switch to the standardized "*mlkem*" algorithms as support for Kyber will be removed with the next release of `liboqs` and `oqsprovider`."

Presence/absence of APIs invisible/inaccessible to users of `oqsprovider` IMO do not need to be documented at all in the release notes.